### PR TITLE
Group members

### PR DESCRIPTION
**Pull request by akaviaLab**
This reverts commit 81717afd

It will fix part of 1214, but not necessarily all of it.

### CODE REVIEW

File: .gitignore
Comment (cdiener) [inline .gitignore]: That seems to be specific to your setup, probably better to just ignore the full `.idea` folder.

**Review by cdiener — COMMENTED** [review]
Makes sense to me but would be good to get a second opinion as well.

File: .gitignore
Comment (akaviaLab) [reply .gitignore]: Yeah. I'm trying, but having problems. If you can ignore this file when merging, that would be great.

**Comment by akaviaLab** [comment]
Ping. Did someone want to review this?

**Comment by cdiener** [comment]
I'll probably merge your MAT PR before this. If you rebase on main afterwards I'll be happy to merge this right after.

**Review by cdiener — APPROVED** [review]
:+1:

**Comment by cdiener** [comment]
Looks, like this doesn't have any conflicts and needs no rebase, so I'll just wait for the CI to finish on the previous merge and merge this one after. Thanks!

**Pull request merged**
